yt/visualization/profile_plotter.py

```diff
@@ -729,15 +729,16 @@ def _get_field_label(self, field, field_info, field_unit, fractional=False):
         if isinstance(field, tuple):
             field = field[1]
         if field_name is None:
-            field_name = r"$\rm{" + field + r"}$"
-            field_name = r"$\rm{" + field.replace("_", r"\ ").title() + r"}$"
+            field_name = field_info.get_latex_display_name()
         elif field_name.find("$") == -1:
             field_name = field_name.replace(" ", r"\ ")
             field_name = r"$\rm{" + field_name + r"}$"
         if fractional:
             label = field_name + r"$\rm{\ Probability\ Density}$"
         elif field_unit is None or field_unit == "":
             label = field_name
+        elif "frac" in field_unit:
```

Comment:
This gets me worried a little if you have a unit name that contains the four letters `frac`. None of them comes to my mind right now, but since users can define custom units, you could imagine something like `fraction_density` or whatever.
Shouldn't this be replaced by
```suggestion
        elif r"\frac" in field_unit:
```
or

```suggestion
        elif r"\dfrac" in field_unit:
```

Reply:
I get by your worrying but I don't understand what any of your alternatives does ?

Reply:
The piece of code is checking that the latex representation is e.g.
```latex
\frac{\rm{g}}{\rm{cm}^{3}}
```
which we can catch by explicitly checking that the string contains the `\frac` substring (or even `\frac{`)

Reply:
oh right, that makes sense